How to build everything.

rokups opened the pull request:
A new page with build instructions. This is a companion PR for https://github.com/rbfx/rbfx/pull/555.

rokups committed:
How to build everything.

gleblebedev commented on pages/documentation/build.md:
```diff
@@ -0,0 +1,257 @@
+
+   This command builds the project.
+
+   Alternatively, you may open the Xcode project a generated  in `cmake-build`` directory.
```
The same is true to VS?

gleblebedev commented on pages/documentation/build.md:
````diff
@@ -0,0 +1,257 @@
+   cmake --build cmake-build-uwp --config Release
+   ```
+
+   This command builds the project in Release mode.
````
VS calls it configuration, not mode. Please rename for consistency.

rokups committed:
Rename "mode" to "configuration".
Add a note about opening visual studio projects.
Fix folder names in the note about opening IDE projects.
